src/nft/components/collection/ActivityCells.tsx

```diff
@@ -65,7 +74,7 @@ const formatListingStatus = (status: OrderStatus): string => {
     case OrderStatus.Expired:
       return 'Expired'
     case OrderStatus.Valid:
-      return 'Add to Bag'
+      return 'Unavailable'
```

Comment:
nit: this could definitely be confusing to an eng coming in, could you add a comment explaining that if we reach this text it means that this order isn't purchaseable?

another alternative is we call `isPurchasableOrder` only in `formatListingStatus` and also pass in `isSelected` this would allow for all potential status text to originate from this fn.